fvtest/porttest/CMakeLists.txt

```diff
@@ -136,16 +136,23 @@ if(OMR_HOST_OS STREQUAL "aix")
 	)
 endif()
 
+# Some tests are known to file when running under simulator such as QEMU.
+# Disable them here.
+set(porttest_filter "*")
+if(CMAKE_CROSSCOMPILING)
+	set(porttest_filter "-PortFileTest2.*:PortMmapTest.*:PortSigTest.*:PortSysinfoTest.*")
+endif()
+
 if (NOT OMR_HOST_OS STREQUAL "aix" AND NOT OMR_HOST_OS STREQUAL "zos")
```

Comment:
Honestly I don't think we should really care too much about cross compiled porttests. I would just add `AND NOT CMAKE_CROSSCOMPILING` to the `if `on line 146 and not run it at all. We get coverage for port test on other platforms anyway.